docs/assets/diagrams/linking/3-unlinking.puml

```diff
@@ -0,0 +1,66 @@
+@startuml Unlinking
+
+title: Unlinking
+hide footbox
+
+box "Mobile Device" #Light
+  actor Alice
+  participant "PISP App" as App
+end box
+box "PISP" #Light
+  participant "Firebase" as Firebase
+  participant "PISP Server" as Server
+end box
+box "Mojaloop" #Light
+  participant "Switch" as Mojaloop
+end box
+
+'start flow
+Alice -> App++: I want to unlink this bank account
+
+App -> Server ++: ""**DELETE /consents/123**""
+App <-- Server: ""202 Accepted""
+deactivate App
+
+Server -> Adapter ++: ""**DELETE /consents/123**""
+Adapter <-- Server: ""202 Accepted""
+deactivate Server
+
+rnote left of Adapter #Light
+  ""**PATCH /consents/123**""
+  ""{""
+  ""  status: "REVOKED",""
+  ""  revokedAt: "2011-10-05T14:48:00.000Z"""
+  ""}""
+end rnote
+Adapter -> Server++: ""**PATCH /consents/123**""
+Adapter <-- Server: ""200 OK""
+deactivate Adapter
+
+rnote left of Server #Light
+  ""firebase.firestore()""
+  ""  .collection("consents")""
+  ""  .where("consentId", "==", "123")""
+  ""  .update(<snapshot object>)""
```

Comment:
```suggestion
  ""  .update({""
  ""    revokedTimestamp: "2011-10-05T14:48:00.000Z",""
  ""    status: "REVOKED"""
  ""  })""
```

I believe it should be something like this

Reply:
wait now im confused
i think we're misaligned cause I thought you wanted this to snapshot object

Reply:
only for the `event(...)`